src/gui/wizard/owncloudsetuppage.cpp

```diff
@@ -269,7 +250,7 @@ bool OwncloudSetupPage::validatePage()
         QString u = url();
         QUrl qurl(u);
         if (!qurl.isValid() || qurl.host().isEmpty()) {
-            setErrorString(tr("Invalid URL"), false);
+            setErrorString(tr("Server address does not seem to be valid"), false);
```

Comment:
@FlexW Really up to you to decide, but, I think, having it as simple as _Server address is invalid_ would do the trick :)

Reply:
Was a suggestion by @jancborchardt . I'm fine with both:) Jan what do you think?